_posts/2015-07-20-why-using-default-scope-is-a-bad-idea.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Why using default_scope is a bad idea"
+date: 2015-07-20 09:55:00
+categories: ["ruby", "rails", "best practices"]
+author: "mauro-oto"
+---
+
+`default_scope` is a method provided by ActiveRecord, which allows you to set
+a default scope (as its name implies) for all operations done on a given model.
+It can be useful for allowing soft-deletion in your models, by having a
+`deleted_on` column on your model and default scoping the model to
```

Comment:
@mauro-oto I'm not sure if "default scoping" is proper English. Maybe you could say "setting the model's default scope to"